demos/node/fonts/load.js

```diff
@@ -2,7 +2,7 @@
 //
 // This demo shows how to use node-canvas to load a font (woff / otf).
 // It does not currently use VexFlow.
-// TODO: Check if the text fonts (PetalumaScript & Roboto Slab) are automatically
+// TODO: Check if the text fonts (PetalumaScript & RobotoSlab) are automatically
```

Comment:
Are we using spaces for both Petaluma Script and Roboto Slab?

Reply:
done